### ISSUE TITLE
Confusing LoggingMiddleware log

### ISSUE THREAD

**Issue by GRouslan** [alexeieleusis/greencat#19]
In logging_middleware.dart lines
`_logger..fine('action: $action')..fine(
            'prev state: ${api.state}')..fine('next state: ${api.state}')..fine(
            '=== === === === === === === === === === === === === === ');`
result to logging same state as prev and next state. ${api.state} for prev state and same  ${api.state} for next state.

**Comment by alexeieleusis** [comment]
You are right! Thanks for the report, this is a fairly easy change, will try to fix it this evening.
